x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -93,6 +93,14 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
     const getTestId = useTestIdGenerator(dataTestSubj);
     const [isVisited, setIsVisited] = useState(false);
 
+    const enterVisitedState = useCallback(() => {
```

Comment:
Maybe call this `refreshVisitedState`, cause that's what this is doing. No?

Reply:
You are right! It's not only about entering the visited state, but also propagating the event upwards by calling `onVisited()`, so I'll just go with the universal `handleVisited()` name which harmonizes with `onVisited()`.